resources/js/components/ui/Table/tquery_filters/DateTimeFilterControl.tsx

```diff
@@ -0,0 +1,117 @@
+import {NON_NULLABLE, cx, useLangFunc} from "components/utils";
+import {BoolOpFilter, DateColumnFilter, DateTimeColumnFilter} from "data-access/memo-api/tquery";
+import {Component, Show, createMemo} from "solid-js";
+import {FilterControlProps} from ".";
+import {tableStyle as ts} from "..";
+
+const OPS = [">=", "<="] as const;
+
+type RangeOp = (typeof OPS)[number];
+
+interface RangeSideFilter extends DateColumnFilter, DateTimeColumnFilter {
+  type: "column";
+  op: RangeOp;
+}
+
+export interface DateTimeRangeFilter extends BoolOpFilter {
+  op: "&";
+  val: RangeSideFilter[];
+}
+
+interface Props extends FilterControlProps<DateTimeRangeFilter> {
+  /** The data type, datetime by default. */
+  columnType?: "date" | "datetime";
+}
+
+export const DateTimeFilterControl: Component<Props> = (props) => {
+  const t = useLangFunc();
+  const inputType = () => (props.columnType === "date" ? "date" : "datetime-local");
+  function toInputValue(date: Date | undefined) {
+    return (
+      date
+        ?.toLocaleString("sv")
```

Comment:
szwedzki❓

Reply:
Tak się składa że działa, zwraca prawie ISO string czasu lokalnego. Sam tego nie wymyśliłem.

Reply:
spoko, też widziałem to u mnie w firmie, dobrze wiedzieć

akurat tam gdzie to widziałem, nie działało to tak jak powinno, ale tu pewnie działa :D